docs/sources/reference/cli/run.md

```diff
@@ -38,6 +38,7 @@ The following flags are supported:
 * `--server.http.memory-addr`: Address to listen for [in-memory HTTP traffic][] on (default `alloy.internal:12345`).
 * `--server.http.listen-addr`: Address to listen for HTTP traffic on (default `127.0.0.1:12345`).
 * `--server.http.ui-path-prefix`: Base path where the UI is exposed (default `/`).
+* `--server.http.live-debugging-buffer-stream-size`: Buffer stream size used for buffering the live debugging entries (default `1000`)
```

Comment:
a command flag is not super ideal because you need to restart Alloy to update the size of the buffer. I would recommend adding this functionality as an attribute of the [live debugging block](https://grafana.com/docs/alloy/latest/reference/config-blocks/livedebugging/). This way you can just update the size of the buffer in the config if you see that it's not enough and hit the reload endpoint (localhost:12345/-/reload)

Reply:
Yup makes sense have made the changes